packages/tools/generator-pie-component/src/app/templates/pie-placeholder.__stories__.ts

```diff
@@ -1,21 +1,24 @@
 import { html, TemplateResult } from 'lit';
-import type { StoryObj as Story } from '@storybook/web-components';
+import { type StoryObj as Story } from '@storybook/web-components';
 import { Pie<%= componentName %>, <%= componentName %>Props } from '@justeattakeaway/pie-<%= fileName %>';
-import { StoryMeta } from '../types';
+import { type StoryMeta } from '../types';
+<% if (needsRTL) { %>import { i18nArgTypes } from '../args/commonArgsTypes';<% } %>
 
 // TODO: Remove this const when other exports from Pie<%= componentName %> are used on Stories, 
 // otherwise tree-shaking will get rid of the web component definition
 const keptReference = Pie<%= componentName %>;
 
 type <%= componentName %>StoryMeta = StoryMeta<<%= componentName %>Props>;
 
-const defaultArgs: <%= componentName %>Props = {};
+const defaultArgs: <%= componentName %>Props = {
+    <% if (needsRTL) { %>dir: 'rtl',<% } %>
```

Comment:
I think this one should default to LTR e.g `dir: 'ltr'`.

Reply:
updated!